src/components/App.js

```diff
@@ -10,6 +11,7 @@ import {
 } from 'react-router-dom';
```

Comment:
separate external / internal packages. so put react-router-dom under the 'react' import and put an extra line beneath it